Documentation/network/servicemesh/installation.rst

```diff
@@ -6,6 +6,13 @@ Prerequisites
   for more details.
 * The minimum supported Kubernetes version for Ingress is 1.19.
 
+.. Note::
+    The ``Ingress`` feature in Cilium require the layer 7 proxy to be enabled in order to function properly. 
+    If the layer 7 proxy is not enabled, this feature will not work as intended. 
+    This feature is enabled by default but can explicitly be enabled with ``--enable-l7-proxy`` flag.
+    Before using the ``Ingress`` feature, ensure that your Cilium installation has the layer 7 proxy enabled and properly configured. If you have any questions or concerns, 
+    consult the Cilium documentation or reach out to the `Cilium community <https://cilium.herokuapp.com/>`_ for support.
+
```

Comment:
This one will also show up in page `L7-Aware Traffic Management`, which also includes this file, see [the preview](https://deploy-preview-23756--docs-cilium-io.netlify.app/network/servicemesh/l7-traffic-management.html). But on that page the requirement is not relevant, and worse, it's confusing to have that block talking about Ingress.

That's why I suggested to split the requirements between the two files.

Reply:
Alright